src/main/kotlin/app/revanced/patches/youtube/layout/crowdfundingbox/patch/CrowdfundingBoxPatch.kt

```diff
@@ -47,7 +47,7 @@ class CrowdfundingBoxPatch : BytecodePatch(
         val crowdfundingBoxResult = CrowdfundingBoxFingerprint.result!!
         val crowdfundingBoxMethod = crowdfundingBoxResult.mutableMethod
 
-        val moveResultObjectIndex = crowdfundingBoxResult.scanResult.patternScanResult!!.endIndex + 2
+        val moveResultObjectIndex = crowdfundingBoxResult.scanResult.patternScanResult!!.endIndex
 
```

Comment:
Rename this to insertIndex and add +1 to it instead of where you use the variable.

Reply:
> Rename this to insertIndex and add +1 to it instead of where you use the variable.

This field is also used to get the move-result-object register index. So if I put +1 there, I'll must put -1 on the register getter.

This change is strictly needed?

Reply:
<img width="669" alt="Immagine 2022-10-25 175058" src="https://user-images.githubusercontent.com/108184954/197821649-e223893a-ade2-4243-ace0-ab120a216a66.png">

Reply:
No, looks good.

Reply:
I'd personally create a separate variable for insertIndex in that case for readability.

Reply:
> I'd personally create a separate variable for insertIndex in that case for readability.

Ok, I'll do it now.

Reply:
> I'd personally create a separate variable for insertIndex in that case for readability.

Done